accounts/decorators.py

```diff
@@ -0,0 +1,42 @@
+from django.http import HttpResponseForbidden
+from functools import wraps
+from django.shortcuts import get_object_or_404
+from hw.models import Assignment, Homework
+
+def teacher_required(view_func):
+     @wraps(view_func)
+     def _wrapped_view(request,*args,**kwargs):
+          if request.user.is_authenticated and request.user.is_teacher:
+               return view_func(request,*args,**kwargs)
+          return HttpResponseForbidden("nemáte přístup k této stránce-dekorator.")
```

Comment:
Chyba zobrazená uživateli by měla popisovat smysl toho `teacher_required`. Naopak to, že je to dekorátor, uživatele vůbec nemusí zajímat :blush: